src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -1311,35 +1311,34 @@ protected override void GetItem(string path)
                     // If we want to retrieve the file streams, retrieve them.
                     if (retrieveStreams)
                     {
-                        if (!isContainer)
+                        foreach (string desiredStream in dynamicParameters.Stream)
                         {
-                            foreach (string desiredStream in dynamicParameters.Stream)
+                            // See that it matches the name specified
+                            WildcardPattern p = WildcardPattern.Get(desiredStream, WildcardOptions.IgnoreCase | WildcardOptions.CultureInvariant);
+                            bool foundStream = false;
+
+                            foreach (AlternateStreamData stream in AlternateDataStreamUtilities.GetStreams(result.FullName))
                             {
-                                // See that it matches the name specified
-                                WildcardPattern p = WildcardPattern.Get(desiredStream, WildcardOptions.IgnoreCase | WildcardOptions.CultureInvariant);
-                                bool foundStream = false;
+                                if (!p.IsMatch(stream.Stream))
+                                { continue; }
 
-                                foreach (AlternateStreamData stream in AlternateDataStreamUtilities.GetStreams(result.FullName))
-                                {
-                                    if (!p.IsMatch(stream.Stream)) { continue; }
+                                string outputPath = result.FullName + ":" + stream.Stream;
+                                // Alternate data streams can never be containers.
+                                WriteItemObject(stream, outputPath, false);
```

Comment:
```suggestion
                                WriteItemObject(stream, outputPath, isContainer: false);
```

Reply:
@kyanha I'm not seeing this change made and there's no discussion about it, is there a reason it was marked as resolved?

Reply:
I had thought I'd used the "Commit suggestion" button from Github's interface.  Let me see what's going on here.